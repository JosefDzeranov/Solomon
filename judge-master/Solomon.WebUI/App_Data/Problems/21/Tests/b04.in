10 10
7 8
1 9
3 4
5 6
8 2
2 10
10 1
4 5
6 7
9 3
