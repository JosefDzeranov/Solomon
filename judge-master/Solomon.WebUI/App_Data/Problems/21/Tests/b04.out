9
3 9 9 1
4 3 3 1
5 4 4 1
6 5 5 1
7 6 6 1
8 7 7 1
2 8 8 1
10 2 2 1
1 10 10 1
